packages/common/data/modern-browsers.json

```diff
@@ -1,9 +1,11 @@
 {
-  "edge": "16",
-  "firefox": "60",
-  "chrome": "61",
-  "safari": "10.1",
-  "opera": "48",
-  "ios_saf": "10.3",
-  "and_ff": "60"
+  "Edge": "16",
+  "Firefox": "60",
+  "Firefox Mobile": "60",
```

Comment:
Does ua-parser-js support `Firefox Mobile` and `Chrome Headless` ? Because I don't see them in [`Possible 'browser.name'`](https://github.com/faisalman/ua-parser-js#methods)

Reply:
@clarkdo maybe we should use engine instead of browsers? There a lot of webkit based browsers, e.g. yandex, vivaldi etc.  While old browserslist-useragent was working for yandex, this new code will require adding Yandex to this list, as well as Vivaldi ( wasnt working before too )

Reply:
@clarkdo Good catch!
- ["Firefox" takes care of mobile as well](https://github.com/faisalman/ua-parser-js/blob/master/src/ua-parser.js#L350) so "Firefox Mobile" can be removed for now.  There is a [PR up for that](https://github.com/faisalman/ua-parser-js/pull/251).
- ["Chrome Headless" is taken care of](https://github.com/faisalman/ua-parser-js/blob/master/src/ua-parser.js#L325)

@aldarund Should we change it to engine in this PR or address it in another?

Reply:
Actually it wont work, user agent report layout version and as i see from https://en.wikipedia.org/wiki/Google_Chrome_version_history
and it dont change from chrome 27 till now and we need javascript engine version and its tied to browser version itself